syntax/erlang.vim

```diff
@@ -125,11 +125,19 @@ syn match erlangNumberFloat   '\<\d\+\.\d\+\%([eE][+-]\=\d\+\)\=\>'
 
 " Strings, atoms, characters
 syn region erlangString            start=/"/ end=/"/ contains=erlangStringModifier
+syn region erlangStringTripleQuoted matchgroup=String start=/"""/ end=/\%(^\s*\)\@<="""/ keepend
+
+" Documentation
+syn cluster erlangDocStringContains contains=erlangInnerDocAttribute,erlangDocStringDelimiter
```

Comment:
I suggest naming this `erlangDocStringCluster` (since it's a `syntax cluster`).

Also, I suggest moving it below the four `erlangDocString` lines. That way we keep a top-down order and it's easier to understand this block (`erlangDocString` contains `erlangDocStringCluster` , which contains `erlangDocStringDelimiter`).